pages/[...slug].vue

```diff
@@ -3,32 +3,41 @@
     <NuxtLayout :name="layout">
       <main>
         <SiteBreadcrumbs />
-        <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
-        <component :is="renderCustomElements(page.content)" />
+        <DrupalTabs v-if="page?.local_tasks" :tabs="page.local_tasks" />
+        <component :is="renderCustomElements(page.content)" v-if="page?.content"  />
```

Comment:
this makes no sense. the page sohuld be able to rely on the page object to be there